src/main/Main.java

```diff
@@ -0,0 +1,51 @@
+package main;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        int elements = 1_000_000;
+
+        System.out.println("\n////////////ArrayLIst////////////\n");
+
+        ListHandler arr = new ListHandler(elements, true);
+        arr.add();
+        arr.remove(500);
+        arr.get(8000);
+
+        System.out.println("\n///////////LinkedList////////////\n");
+
+        ListHandler lin = new ListHandler(elements, false);
```

Comment:
you can use one variable "listHandler" and reuse it for second list
ListHandler listHandler = new ListHandler(elements, true);
...
listHandler = new ListHandler(elements, false);
...